### ISSUE TITLE
A11y_SQL database projects extension_Publish_New SQL Server_ScreenReader : NVDA is announcing unnecessary information as invalid entry.

### ISSUE THREAD

**Issue by AnKushSingh05** [microsoft/azuredatastudio#22220]
### GitHub Tags
#A11yWCAG2.1; #A11yTCS; #A11ySev3; #DesktopApp; #ScreenReader; #WCAG1.3.1; #Win11; #SH-SQL Azure Data Studio-Win32-March23; #Win32; #SQL Azure Data Studio; #NVDA; #AILimited; #Info and Relationships;

### Actual Result:​
NVDA is announcing unnecessary information as invalid entry.

### Expected Result:​
NVDA should not announce unnecessary information as invalid entry.

### Environment Details:
Application Name: SQL Azure Data Studio
Version: 1.41.2 (user setup)
Microsoft Windows 11 Enterprise 10.0.22621 Build 22621
Screen reader: NVDA

### Repro Steps:

1. Launch Azure Data Studio
2. Tab till extension and search for “SQL Database Projects” and install it.
3. Tab till database project and press enter.
4. “Database Project” panel will open.
5. Tab till “View And More Action” button and press enter.
6. Tab till “Create Project From Database” and press enter.
7. Tab till server field and press enter.
8. “Connection” dialog will open. Fill the details for SQL server and connect.
9. Tab till Create button and press enter.
11. Database project will get create and will appear in Database project panel.
12. Tab till created project name in Database Project panel.
13. Press context menu button(right click).
14. Tab till Publish and press enter.
15. Publish Project window will open.
16. Tab till “Publish to a new SQL server local development container” radio button under “Publish Target” and select it.
17. Prerequisite: have Docker installed on your machine.
18. Tab till port (select a port or keep the default ).
19. Enter a password that meets the password complexity guidelines. 
20. Tab till the docker image/ image tag to change it (if wanted). 
21. Tab till “I accept the Microsoft SQL Server License Agreement” checkbox.
23. Verify whether the NVDA is announcing unnecessary information as invalid entry or not.

### User Impact:
Screen reader users will get confused if it is announcing the unnecessary information as invalid entry.

### ![Unnecessary info](https://user-images.githubusercontent.com/124244478/223650895-9a760081-2407-45db-a2c7-96f61987861b.gif)

**Comment by VijayaBharathyA** [comment]
@kisantia 
Similar issue repro in following screen also
### Environment and OS details:
Azure Data Studio Version: 1.42.0 (user setup)
Windows 11 Enterprise Version	22H2
Installed on	‎28-‎12-‎2022
OS build	22621.1413
Experience	Windows Feature Experience Pack 1000.22639.1000.0
NVDA Version 2023.1
JAWS Version 2023
### Pre-Requisite:

1. Install Azure data studio From here https://learn.microsoft.com/en-us/sql/azure-data-studio/download-azure-data-studio?view=sql-server-ver16&tabs=redhat-install%2Credhat-uninstall
2. Once installation done , Install MySQL Extension in Azure data studio
3. Tab Till  new connection and press enter 
4. Enter a valid server name username and password and press connect
### Repro Steps:

1. Launch Azure Data Studio.
2. Turn on NVDA
3. Tab till New connection and Press enter
4. Tab till Server
5.  Verify that Narrator is announcing Invalid entry or not

### Actual Result:
When we navigate to the Server Required field, which is blank, the NVDA announces "Invalid."
**Issue Repro with JAWS Also**
**Notes**
Similar issue repro in Following Screen also
### New Data Base :
1. Launch Azure Data Studio.
2. Connect to server.
3. In the Connections pane, right-click on a server connection. 
4. Then select 'Manage' from the context menu to view the dashboard.
5. From dashboard, click on New Database.
6. Tab till Name and Verify Screen reader is announcing Invalid or not.
### New deployment - Azure Database for MySQL
1. Launch Azure Data Studio.
2. Select the Connections icon (if not already selected) from left bar.
3. Click on the ellipses '...' and select "New Deployment"
4. In the Deployment options, select "Azure Database For MySQL" card.
5.Verify Screen reader is announcing invalid for accept terms of use checkbox or not
### Expected Result:
NVDA should not announce as an invalid entry.
### Attachment:
** Issue :**
![main issue2](https://user-images.githubusercontent.com/100189407/229813030-747b1f02-b384-40c1-ae27-966718f4561c.png)


![Screenshot (74)](https://user-images.githubusercontent.com/100189407/229688666-a111aae2-51ee-47a8-9d06-ca07b031319a.png)


**Notes Issue 1:**
![notes issue 1 nvda](https://user-images.githubusercontent.com/100189407/229813290-b678e530-ef5b-4611-b0cf-61f073bfb467.png)

![Screenshot (78)](https://user-images.githubusercontent.com/100189407/229688862-b4451545-afd2-444a-b742-1dd799d7bae8.png)

**Notes Issue 2:**
![Screenshot (59)](https://user-images.githubusercontent.com/100189407/229813629-0db495d9-7043-44e3-8bba-7cb0a26f9f28.png)

![Screenshot (75)](https://user-images.githubusercontent.com/100189407/229688660-46026e83-364c-482f-ba25-cceaa36f7abc.png)

**Comment by VijayaBharathyA** [comment]
Primary Bug :https://github.com/microsoft/azuredatastudio-mysql/issues/192
